api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "unversioned"
+   "version": "v1.6.0"
```

Comment:
I still think it should be a binary version and I am in favor of having the short version in swagger.json. I don't expect people to use half-baked under-development swagger.json for anything serious.